packages/desktop/views/dashboard/send-flow/views/SelectTokenView.svelte

```diff
@@ -58,9 +62,9 @@
     function onContinueClick(): void {
         setNewTransactionDetails({
             type: NewTransactionType.TokenTransfer,
-            assetId: selectedAssetId,
+            assetId: selectedAsset.id,
```

Comment:
why not store the whole asset if we might need it in multiple places in the flow

Reply:
This might break things for mobile. E.g. in `AmountView`.